build.py

```diff
@@ -1679,7 +1679,9 @@ def create_docker_build_script(script_name, container_install_dir, container_ci_
             if FLAGS.container_memory:
                 baseargs += ["--memory", FLAGS.container_memory]
 
-        baseargs += ["--cache-from={}".format(k) for k in cachefrommap]
+        if target_platform() != "windows":
```

Comment:
Won't this change adversely impact Triton users who do not have access to the prebuilt ORT binaries?

Reply:
Not it shouldn't. 
1. `--cache-from` flag given to top level docker process, not to Docker-out-of-Docker.
2. Preload cache on our runners take about 10-15 minutes.

We can drop this change if your concerns are adamant.

Reply:
I'm probably fine with the change, just trying to understand. I'm imagining a scenario where a user is attempting to do a rebuild of Triton, in which case, I'd assume utilizing these caches would speed up their rebuild time.

Seems like the main benefit from our end is that it speeds our builds up by 10-15 minutes, though?

Reply:
> Seems like the main benefit from our end is that it speeds our builds up by 10-15 minutes, though?
Do you have a confirmation that it speeds up our build?

Reply:
I mean, it's a build cache--there wouldn't be much point of having it if it didn't speed up the build. I don't have the cycles to confirm via experimentation.

All that said, I think most Windows users download the release asset and do not build from scratch. Additionally, if this would improve our CI build times I view it as a net win. If build performance issues are reported, it's simple to revert.